app_test.go

```diff
@@ -1401,6 +1401,16 @@ func Test_App_Master_Process_Show_Startup_MessageWithAppName(t *testing.T) {
 	utils.AssertEqual(t, true, strings.Contains(startupMessage, app.Config().AppName))
 }
 
+func Test_App_Master_Process_Show_Startup_MessageWithAppNameNonAscii(t *testing.T) {
```

Comment:
Can you move all startup message, route list tests to listen_test.go

Reply:
> Can you move all startup message, route list tests to listen_test.go

Sure